opencog/nlp/learn/README

```diff
@@ -418,38 +409,43 @@ Minimum spanning tree code is in `mst-parser.scm`. The current version
 works well.
 
 To perform MST parsing in bulk:
-* Copy `run/mst-count-en.scm`, `run/mst-one.sh` and `run/wiki-mst-en.sh`
-  to your working directory.
+* Review the file `opencog/nlp/learn/run/README`, and then copy the
+  scripts into your working dir, if you didn't do it before.
 
 * (Optional but suggested) Make a copy of your word-pair database, "just
   in case".  You can copy databases by saying:
 ````
-  createdb -T existing_dbname  new_dbname
+  createdb -T existing_dbname new_dbname
 ```
 
-* Edit `mst-count-en.scm` and set the correct database login
-  credentials. (Maybe to the new_dbname you crate above.)
-
 * Edit `wiki-mst-en.sh` and change the directory `gamma-pages` to
   whatever directory holds your source text (or copy your text to
   the `gamma-pages` directory). Be aware that, during processing,
   text files are removed from this directory.
 
-* Start the REPL server by running ./run-all-servers.sh or
+* If you are interested in the actual sentence parses, open
+  `mst-one.sh` and change the observe variable from `observe-mst` to
+  `observe-mst-extra`. Parses will be written in `mst-parses.txt`.
+
+* Start the REPL server by running 
 ```
-  guile -l mst-count-en.scm
+  guile -l parse-launch.scm -- --db new_dbname --user your_user --password asdf
```

Comment:
Parsing was done in the earlier step. This is the MST step, not the parse step.   Call it "mst-launch" or something like that.